### PR TITLE
Package K8S/EKS

### PR DESCRIPTION
**Pull request by yorinasub17**
Put info about our K8S/EKS package development on the IaC library.

### CODE REVIEW

File: _data/library-infrastructure-packages.yml [hunk 1]
```diff
@@ -249,3 +249,18 @@
...
     Create Kinesis streams with configurable or auto-calculated shard and retention settings.
 
+- name: "Kubernetes / EKS"
+  tags:
+    - AWS
+  icons:
+    - title: Terraform
+      image: icon-terraform.png
+    - title: Python
+      image: icon-python.png
```
Comment (josh-padnick) [inline _data/library-infrastructure-packages.yml]: Not a blocker for this PR, but it occurs to me that someday we should have k8s logo or tag so we can mark which of our packages support k8s.

File: pages/infrastructure-as-code-library/_library.html [hunk 2]
```diff
@@ -7,6 +7,9 @@ <h2 id="infra-modules" class="margin-top-none">Infrastructure Modules</h2>
           <div>
             <h4 class="margin-top-none">{{ package.name }}</h4>
             <p>{{ package.description }}</p>
+            {% if package.private_beta %}
+            <p><small><em>This package is in private beta.<br/>Contact support@gruntwork.io for more information.</em></small></p>
+            {% endif %}
```
Comment (josh-padnick) [inline pages/infrastructure-as-code-library/_library.html]: Nice touch.

File: _data/library-infrastructure-packages.yml (hunk 1 above)
Comment (yorinasub17) [reply _data/library-infrastructure-packages.yml]: Oh I was actually thinking that it belongs on the cloud, but that makes sense too!

File: pages/infrastructure-as-code-library/_library.html (hunk 2 above)
Comment (brikis98) [reply pages/infrastructure-as-code-library/_library.html]: +1

**Comment by brikis98** [comment]
Reminder: our website does not auto deploy. You have to deploy it manually as defined in the README.

**Pull request merged**